src/spec/settings_spec.js

```diff
@@ -132,4 +132,29 @@ describe('Settings Page', () => {
     expect(settings.getDefaultDomain()).not.toEqual(defaultDomain);
     expect(settings.getDefaultDomain()).toEqual("fromdoppler.com");
   });
+
+  it('should set as enabled domain correctly', () => {

```

Comment:
The test it's for activate a Domain disabled, maybe "Should activate the domain correctly"=